main.go

```diff
@@ -348,6 +348,7 @@ func {{.FunctionPrefix}}FSByte(useLocal bool, name string) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
+		defer f.Close()
```

Comment:
isn't it the same but w/o having to store the result of ReadAll and check for the error? meh though it's less code like this